build.gradle

```diff
@@ -114,7 +114,7 @@ task allPlatform(type: Zip, group: 'release', dependsOn: [renameShadowJar, updat
 		into('icons')
 		exclude("icon.icns")
 	}
-	['readme.html', 'changelog.txt', 'system.ini', 'TripleA_RuleBook.pdf', 'mapDownload.properties',
```

Comment:
Renaming 'mapDownload.properties' to 'game_engine.properties', and consolidating game config to the one file.